test/e2e/adminUI/pages/fieldTypes/datetime.js

```diff
@@ -0,0 +1,56 @@
+var utils = require('../../../utils');
+
+module.exports = function DatetimeType(config) {
+	var self = {
+		selector: '.field-type-datetime', //There should be a [for="' + config.fieldName + '"] here, but this isn't in the html.
```

Comment:
Line 5 causes problems. The selector div should have a `for="fieldA` in the html code, however it does not. 

This means the tests pass for fieldA, since it is the first instance of a datetime div in the html, however the tests fail for fieldB as it's looking for fieldB stuff in the wrong `field-type-datetime` div.